packages/friends/yml/en/daniel-wheeler.yml

```diff
@@ -1,7 +1,18 @@
 name: Daniel Wheeler
 slug: daniel-wheeler
 gender: male
-description: TODO
+description:
+  The life of Daniel Wheeler (1771-1840) is a monument to the heart purifying power of the
+  gospel of Jesus Christ. Left an orphan at a very young age, Wheeler was placed by a
+  friend of the family as an apprentice on a sailing vessel where, along with learning
+  sailing and navigation, he became acquainted with every form of vice and immorality.
+  After six years in the navy, he joined the army until his 26th year, at which time the
+  Lord made use of a hurricane at sea to awaken him to his lost condition. Being
+  mercifully enabled to repent, and also to see the entire spirituality of the Gospel
+  dispensation, he soon left the army and joined the Society of Friends in London. His
+  growth in grace was quick and steady, and after some years he became a minister of great
+  esteem, and was eventually led by the Lord to preach the gospel of Christ in a multitude
```

Comment:
I made both of the changes you suggested on this one. It's still kinda long, but I think you can make it work by tweaking the CSS, because its a large book.